modules/client-configuration/pages/clients-rh.adoc

```diff
@@ -252,6 +260,13 @@ From the command prompt, add [systemitem]``rhel-x86_64-server-6`` and [systemite
 From the {webui}, add [systemitem]``RHEL7 Base x86_64`` and [systemitem]``SUSE Linux Enterprise Client Tools RES7 x86_64``.
 +
 From the command prompt, add [systemitem]``rhel-x86_64-server-7`` and [systemitem]``res7-suse-manager-tools-x86_64``.
++
+* For {es} 8:
++
+From the {webui}, add [systemitem]``RHEL8 Base x86_64`` and [systemitem]``SUSE Linux Enterprise Client Tools RES8 x86_64``.
+// check what the correct name for the AppStream channel is. LKB 2019-12-13
```

Comment:
Now I know what you mean. The Appstream channel does NOT come from SUSE. It needs to be imported from the DVD, so you need to add the instructions below.

Reply:
Do we still need these instructions?

```
mkdir rhel8
mkdir iso_rhel8
cd iso_rhel8/
wget http://schnell.suse.de/RHEL/RHEL-8/rhel-8.0-x86_64-dvd.iso
cd iso_rhel8/
mount -o loop /iso_rhel8/rhel-8.0-x86_64-dvd.iso /rhel8/
cd /var/lib/rpm
spacewalk-repo-sync -c rhel8-iso-base_os -u file:///rhel8/BaseOS/
cd /rhel8/
# wait for sync to complete...
cd /
cd /var/lib/rpm
spacewalk-repo-sync -c rhel8-iso-app_stream -u file:///rhel8/AppStream/
#wait for sync to complete...
mgr-create-bootstrap-repo --with-custom-channels
1. RHEL8-x86_64
2. SLE-ES8-x86_64
Enter a number of a product label: 1
Creating bootstrap repo for latest Service Pack of RHEL8-x86_64
```

Reply:
We are mixing things. Those instructions are NOT for RHEL, are for SLES ES. The instructions at https://github.com/uyuni-project/uyuni-docs/pull/27/files#diff-f1d37138af48a9d14c0950047b0a724dR164 are still NOT updated.

Reply:
We can do that after this release goes out, since we need to update these sections in their entirety anyway. For now, the RH page says that readers need to arrange access to their own RH base media.